### PR TITLE
fix(components, protocol-designer): Fix text wrap issues in PD

### PR DESCRIPTION
**Pull request by koji**
# Overview
Fix text wrap issues in PD (not only RQA-3238)

close [RQA-3238](https://opentrons.atlassian.net/browse/RQA-3238)

## Risk assessment
low

[RQA-3238]: https://opentrons.atlassian.net/browse/RQA-3238?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: protocol-designer/src/pages/ProtocolOverview/index.tsx [hunk 1]
```diff
@@ -223,8 +223,11 @@ export function ProtocolOverview(): JSX.Element {
   const cancelModal = (): void => {
     setShowExportWarningModal(false)
   }
+
+  console.log('obj', JSON.stringify(allIngredientGroupFields, null, 4))
```
Comment (jerader) [inline protocol-designer/src/pages/ProtocolOverview/index.tsx]:
```suggestion
```

File: protocol-designer/src/pages/ProtocolOverview/index.tsx [hunk 2]
```diff
@@ -223,8 +223,11 @@ export function ProtocolOverview(): JSX.Element {
...
+
   return (
-    <>
+    <Fragment>
```
Comment (jerader) [inline protocol-designer/src/pages/ProtocolOverview/index.tsx:228]: why do we need to specify that this is a fragment here?

**Review by jerader — APPROVED** [review]
nice! left some comments

File: protocol-designer/src/pages/ProtocolOverview/index.tsx (hunk 2 above)
Comment (koji) [reply protocol-designer/src/pages/ProtocolOverview/index.tsx:228]: it's getting a warning because of the recently eslint change.

**Pull request merged**